Vulnerabilities/ScreenConnect/ScreenConnect.07.0 - Identify shells being spawned from ScreenConnect.sql

```diff
@@ -0,0 +1,87 @@
+/*************************** Sophos.com/RapidResponse ***************************\
+| DESCRIPTION                                                                    |
+| Identify shells being spawned from ScreenConnect process.                      |
+|                                                                                |
+| IMPORTANT                                                                      |
+| If the description field results shows 'Cloud' then the process command lines  |
+| being spawned are from a cloud managed ScreenConnect instance that would have  |
+| been patched already by ConnectWise.                                           |
+|                                                                                |
+| If the description field results shows 'Self-Hosted' then the command lines    |
+| being spawned are from a self-hosted version of ScreenConnect that is          |
+| potentially vulnerable and may have been exploited.                            |
```

Comment:
We can note the instance_url field provides the name of the On-Prem Server. Helpful to locate it like the IP in the other query.